### ISSUE TITLE
Harmonize templating mechanisms

### ISSUE THREAD

**Issue by huard** [CSHS-CWRA/RavenPy#31]
## Description

At the moment, there are two mechanisms to write configuration files. 

### Legacy 
Read existing template file including named formatting tags, e.g. `{elevation}`, then use attributes and properties in RV instances to return values (converted by format) or strings for more elaborate constructs.

### New
Classes with `to_rv` method explicitly describing the conversion to configuration file content. 

I think we can probably bring the two approaches together by having a base class: 
```
class Command(NamedTuple):
    def __format__(self, format_spec):
        return self.to_rv()
```
that Raven commands inherit from.

**Comment by cjauvin** [comment]
My current thinking is that it's better to get rid of the `Command` parent class and replace `to_rv()` by `__repr__` everywhere.

**Comment by huard** [comment]
Closed by #30
